### PR TITLE
Update Regex for disk collector to remove nightly,weekly,central disks

### PR DESCRIPTION
**Pull request by tuxgoose**
**Description**

Changes proposed in this pull request:
- updates regex in kyma-integration for diskcollector to remove additional disks
- adds preset-kyma-keyring for encryption(change is in additional pr awaiting review)

**Related issue(s)**

#828

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -268,6 +268,11 @@ presets:
           secretKeyRef:
             name: nightly-aks-github-integration-app-client-secret
             key: client-secret
+  - labels:
+      preset-kyma-keyring: "true"
+    env:
+      - name: KYMA_KEYRING
+        value: "kyma-prow"
```
Comment (tehcyx) [inline prow/config.yaml]: You don't use this anywhere, I think it can be removed?

File: development/tools/jobs/kyma/kyma_integration_test.go [hunk 2]
```diff
@@ -242,7 +242,7 @@ func TestKymaIntegrationJobPeriodics(t *testing.T) {
 	tester.AssertThatHasExtraRefs(t, disksCleanerPeriodic.JobBase.UtilityConfig, []string{"test-infra", "kyma"})
 	assert.Equal(t, "eu.gcr.io/kyma-project/prow/buildpack-golang:0.0.1", disksCleanerPeriodic.Spec.Containers[0].Image)
 	assert.Equal(t, []string{"bash"}, disksCleanerPeriodic.Spec.Containers[0].Command)
-	assert.Equal(t, []string{"-c", "development/disks-cleanup.sh -project=${CLOUDSDK_CORE_PROJECT} -dryRun=false -diskNameRegex='^gke-(gkeint|gke-upgrade).*[-]pvc[-]'"}, disksCleanerPeriodic.Spec.Containers[0].Args)
+	assert.Equal(t, []string{"-c", "development/disks-cleanup.sh -project=${CLOUDSDK_CORE_PROJECT} -dryRun=false -diskNameRegex='^gke-gkeint|gke-upgrade|weekly|nightly|gke-central' -ageInHours=100"}, disksCleanerPeriodic.Spec.Containers[0].Args)
```
Comment (tehcyx) [inline development/tools/jobs/kyma/kyma_integration_test.go]: While this solves a problem every ~5 days now, we could run into creating a lot of disks in the short term. And in the meantime ramp up costs on disks that should only be around for ~3 hours. These disks would now, if not deleted by the resource creating it dangle for ~5 days before getting deleted. Maybe we handle weekly different than the rest.

File: prow/jobs/kyma/kyma-integration.yaml [hunk 3]
```diff
@@ -322,7 +322,7 @@ periodics:
       - "bash"
       args:
       - "-c"
-      - "development/disks-cleanup.sh -project=${CLOUDSDK_CORE_PROJECT} -dryRun=false -diskNameRegex='^gke-(gkeint|gke-upgrade).*[-]pvc[-]'"
+      - "development/disks-cleanup.sh -project=${CLOUDSDK_CORE_PROJECT} -dryRun=false -diskNameRegex='^gke-gkeint|gke-upgrade|weekly|nightly|gke-central' -ageInHours=100"
```
Comment (tehcyx) [inline prow/jobs/kyma/kyma-integration.yaml]: see above comment on the test.

File: prow/config.yaml (hunk 1 above)
Comment (tuxgoose) [reply prow/config.yaml]: this is what I get for using master, this is part of another pr not yet merged

File: development/tools/jobs/kyma/kyma_integration_test.go (hunk 2 above)
Comment (tuxgoose) [reply development/tools/jobs/kyma/kyma_integration_test.go]: are you suggesting we lower this to two hours?

**Pull request merged**